dev-tools/releng/src/main.rs

```diff
@@ -454,8 +454,12 @@ async fn main() -> Result<()> {
                     artifacts_path.as_str(),
                     "target",
                     "create",
+                    "--preset",
+                    target.as_str(),
                 ])
-                .args(target.target_args())
+                // Note: Do not override the preset here! All release targets
+                // must be configured entirely via the `target.preset` table
+                // in `package-manifest.toml`.
```

Comment:
This comment is about the `--preset` arg value above, not the `env_remove` call, right?

Reply:
Yes. I'll try making this clearer.